source/opt/merge_return_pass.cpp

```diff
@@ -111,7 +111,7 @@ bool MergeReturnPass::ProcessStructured(
   }
 
   RecordImmediateDominators(function);
-  AddDummySwitchAroundFunction();
```

Comment:
I don't thing that placeholder is the best name for what is happening in merge return.  Could we use "SingleCaseSwitch" instead?  That is much more descriptive.

Reply:
Done